utils/strings.js

```diff
@@ -1,5 +1,24 @@
 const _ = require('lodash');
 
+const reservedWords = [
```

Comment:
[Code Style] I think that we tend to have hard coded const like that capitalized and underscored:
```suggestion
const RESERVED_WORDS = [
```